frontend/util/services-inputs.component.tsx

```diff
@@ -69,11 +67,31 @@ export default React.forwardRef<ServicesRef, IntakeServicesInputsProps>(
 );
 
 const css = `
-& h1 {
-  font-size: 1.8rem;
-}
+	& h3 {
+		font-size: 1.8rem;
+	}
+
+	& .vertical-options {
+		display: block;
+	}
+
+	& .vertical-options > * {
+		padding: .8rem 0;
+	}
+
+	& .vertical-options > div {
+		display: flex;
+		flex-direction: column;
+		justify-content: flex-start;
+		align-items: flex-start;
+	}
 `;
 
+export type IntakeService = {
+  id: number;
+  serviceName: string;
```

Comment:
I see you moved this from the other file, but forgot to bring over the `serviceDescription` property. Was that on purpose?

Reply:
Yes, we didn't seem to use it anywhere, always the serviceName. Did I miss a spot where we used it?

Reply:
fair enough